ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -488,3 +488,10 @@ def __iter__(self):
 
     def __mod__(self, value, /):
         return np_frontend.mod(self, value, out=self)
+    
+    def ptp(self, *, axis=None, out=None, keepdims=False):
+        ret = ivy.max(self.ivy_array) - ivy.min(self.ivy_array)
+        if axis==int :
+            return ivy.flatten(self.ivy_array, order="C")
+        else:
+            return np_frontend.ndarray(ret, _init_overload=True)
```

Comment:
Other than that, I don't think you are handling `axis` correctly. From what I understand looking at the docs of the function, you should be finding the min and max across the given axis and then subtracting one from the other.

Reply:
I think your code still has some issues. Please try and run the test locally to debug the function until it passes.

Reply:
Hi @AnnaTz ,
thanks for the review. Just pull others request but it keeps give me error 
"TypeError: _flatten_frontend_return() got an unexpected keyword argument 'available_dtypes'"
What should I input?